### DISCUSSION TITLE
Map gallery icon

### DISCUSSION THREAD

**Discussion by FilipLeitner** [hslayers/hslayers-ng#3442] [Polls]
Map gallery icon we use is the exact same as the one used in connection with compositions - composition panel icon, open composition in add-data panel. It would be a good idea to choose a new one

Cant add images to the poll options so at least here 
![image](https://user-images.githubusercontent.com/44566616/195320014-fa680292-e0b2-415e-bea7-2e0bf936ac0e.png)
![image](https://user-images.githubusercontent.com/44566616/195319786-3220c34e-a432-49d8-a60c-33ac0d6f789e.png)
![image](https://user-images.githubusercontent.com/44566616/195319963-e10c8c6e-54f3-43af-b584-3816d11f4492.png)
![image](https://user-images.githubusercontent.com/44566616/195320060-26bd119a-acda-41e9-826d-d292b6c21103.png)
![image](https://user-images.githubusercontent.com/44566616/195320107-ce5744f2-c987-4f56-87d0-dcc7e7fb9478.png)
![image](https://user-images.githubusercontent.com/44566616/195320152-9fd75fd3-8b3f-44c3-a572-fa13fa9988fb.png)
![image](https://user-images.githubusercontent.com/44566616/195320180-1d6a8980-6055-429f-bd23-4b62ab54f82a.png)

**Comment by jmacura** [comment]
And the winner is...
![image](https://user-images.githubusercontent.com/44566616/195320014-fa680292-e0b2-415e-bea7-2e0bf936ac0e.png)
